IdentityCore/src/cache/token/MSIDAccountCacheItem.h

```diff
@@ -43,6 +43,10 @@
 
 @property (readwrite, nullable) NSDictionary *additionalAccountFields;
 
+@property (readwrite, nullable) NSString *lastModificationTime;
+@property (readwrite, nullable) NSString *lastModificationProcess;
+@property (readwrite, nullable) NSString *lastModificationApp;
```

Comment:
what's the difference between last modification process and app?

Reply:
Process is the Proccess ID number.